mk/check.mk

```diff
@@ -51,6 +51,9 @@ format: fmt format/common
 .PHONY: kube-lint
 kube-lint:
 	find ./deployments/charts -maxdepth 1 -mindepth 1 -type d -exec $(KUBE_LINTER) lint {} \;
+	$(KUBE_LINTER) lint \
+		$(KUMA_DIR)/app/kumactl/cmd/install/testdata/install-control-plane*.golden.yaml \
+		$(KUMA_DIR)/app/kumactl/cmd/install/testdata/install-cp-helm/*.golden.yaml
```

Comment:
I don't think KUMA_DIR is right here as in KM you probably just not want to run this.

Maybe it should be: `find ./app/kumactl/cmd/install/testdata -name '*.golden.yaml -exec $(KUBE_LINTER) lint {} \;` ? Looks like this would work for KongMesh too

Reply:
This runs `kube-lint` for every single file separately as opposed to the glob which runs `kube-lint` once. I'll see if there's another way to do it

Reply:
OK `-exec {} +`